src/components/InfoPanel/InfoPanel.test.js

```diff
@@ -1,29 +1,37 @@
+import { render, screen } from "@testing-library/react";
 import configureStore from "redux-mock-store";
 import { Provider } from "react-redux";
-import { mount } from "enzyme";
 import { MemoryRouter, Route } from "react-router";
 import { QueryParamProvider } from "use-query-params";
-import dataDump from "testing/complete-redux-store-dump";
 
+import { reduxStateFactory } from "testing/redux-factory";
 import TestRoute from "components/Routes/TestRoute";
 
 import InfoPanel from "./InfoPanel";
 
 const mockStore = configureStore([]);
 
 jest.mock("components/Topology/Topology", () => {
-  const Topology = () => <div className="topology"></div>;
+  const Topology = () => <div data-testid="topology"></div>;
```

Comment:
I believe the standard is `data-test`

Reply:
Yeah that is(was?) the standard but RTL has a method `getByTestId` in which it requires `data-testid` vs `data-test`.

Reply:
I see, good to know.

Reply:
Better to follow the testing frameworks standard then ours